packages/E2ETest/windows/ReactUWPTestApp/Assets/TreeDump/masters/ControlStyleRegularBorder.json

```diff
@@ -9,10 +9,10 @@
   "Height": 360,
   "HorizontalAlignment": "Stretch",
   "Margin": "0,0,0,0",
-  "RenderSize": [800, 360],
+  "RenderSize": [759, 360],
```

Comment:
>759 [](start = 17, length = 3)

These smaller widths are expected.  RNTester has a container component that has margins on it, and this pages stretches across the entire available space, which is now smaller.

Reply:
the e2etest app had some code on startup which set the window size to 800x600 I believe, to ensure we get a consistent size. We should do that here too, otherwise the masters may not match if people ever resize/maximize etc

Reply:
That code in the app side is still there.  Also the tree dumper control has code to resize to 800x600.

---
In reply to: [457682484](https://github.com/microsoft/react-native-windows/pull/5555#discussion_r457682484) [](ancestors = 457682484)